tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  purge: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {
+      colors: {
+        'primary': '#D4E09B',
+        'secondary': '#ADB6FF'
+      }
```

Comment:
This can be deleted, it was for the other site theme

Reply:
done